src/policy/policy-inl.h

```diff
@@ -0,0 +1,196 @@
+#ifndef SRC_POLICY_POLICY_INL_H_
+#define SRC_POLICY_POLICY_INL_H_
+
+#include "policy/policy.h"
+#include "node_errors.h"
+#include "env-inl.h"
+
+#include <array>
+#include <string>
+#include <vector>
+
+namespace node {
+namespace policy {
+
+Policy* Policy::GetCurrent(Environment* env) {
+  return GetCurrent(env->context());
+}
+
+Policy* Policy::GetCurrent(v8::Isolate* isolate) {
+  return GetCurrent(isolate->GetCurrentContext());
+}
+
+Policy* Policy::GetCurrent(v8::Local<v8::Context> context) {
+  if (UNLIKELY(context.IsEmpty())) {
+    return nullptr;
+  }
+  if (UNLIKELY(context->GetNumberOfEmbedderDataFields() <=
+               ContextEmbedderIndex::kPolicyIndex)) {
```

Comment:
Ideally, we’d check that this is Node.js context using `Environment::GetCurrent() != nullptr` – in an embedding situation, a context could have more than `kPolicyIndex` fields, but the field might have a completely different meaning there

Reply:
ah nice.. ok, good to know. I had a todo to figure out how this would look in embedded scenarios.